light/rpc/client.go

```diff
@@ -490,6 +503,19 @@ func (c *Client) Tx(ctx context.Context, hash []byte, prove bool) (*ctypes.Resul
 	return res, err
 }
 
+// ProveShares calls rpcclient#ProveShares method and returns an NMT proof for a set
+// of shares, defined by `startShare` and `endShare`, to the corresponding rows.
+// Then, a binary merkle inclusion proof from the latter rows to the data root.
+func (c *Client) ProveShares(
+	ctx context.Context,
+	height uint64,
+	startShare uint64,
+	endShare uint64,
+) (types.SharesProof, error) {
+	res, err := c.next.ProveShares(ctx, height, startShare, endShare)
+	return res, err
```

Comment:
Same for this. The light client here is just trusting whatever the full node responds with. In some of the other methods like `Tx` the light client is actually able to prove that the transaction is committed as well as retrieve the transaction. Can this be done for this method?

Reply:
https://github.com/celestiaorg/celestia-core/issues/924